hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
+
+Right now you find most of Documentation in our
+[wiki](https://wiki.mumble.info).
+Any contributions are welcome, improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+
+The new places for documentation will be here:
+
+- [Mumble Github Repo - Docs](https://github.com/mumble-voip/mumble/tree/master/docs)
+
+- [Website's Github repo - Documentation](https://github.com/mumble-voip/mumble-www/tree/master/hugo/content/documentation)
+
+#### Documentation Issues & Pull requests
+
+- [Documentation Issues on Github](https://github.com/mumble-voip/mumble/labels/documentation)
+
+<!--
+Pull requests and issues:
+
+- [DOCS: Add Userguides to the documentation](https://github.com/mumble-voip/mumble/pull/4628) 
+
+- (Work-in-Progress) [C++ Guidelines & tools](https://github.com/mumble-voip/mumble/discussions/4261)
+
+-->
+
+
+---
+
+<!-- 
+**Ice methods**: 
+<!-- outdated? -->
+<!-- Is this here now: https://www.mumble.info/documentation/slice/1.3.0/html/Murmur.html  ? -->
+<!--
+We have a [generated documentation](http://mumble.sourceforge.net/slice/) for exposed Ice methods. Although probably only developers using the doc will be able to understand it, please give us a note on any inconsistencies, errors or needed clarifications if you find them.
+-->
+
+### External Documentation
+
+- **Wikipedia**: Someone that is not a development team member needs to maintain the [article(s) at Wikipedia](https://en.wikipedia.org/wiki/Mumble_%28software%29).
+
+
+## Website
+
+<!-- 
+Todo:
+Add more instructions?
+Here or at the Mumble-www repo.
+
+Such as:
+-Writing guidelines (markdown etc.)
+-Content (what is planned and needs improvement etc.?) 
+
+-->
+
+This website has a Github repository at [mumble-www](https://github.com/mumble-voip/mumble-www).
+
+It consists of two parts:
+
+- a webserver written in GO
+- Most of the content is build with [hugo](https://gohugo.io/) and written in **Markdown**, **HTML** and some css.
+
+You can:
+
+- report [issues](https://github.com/mumble-voip/mumble-www/issues)
+- create [requests](https://github.com/mumble-voip/mumble-www/issues), 
+- create pull requests for changes to the website
+
+
+
+## Community
+
+Be a part of our community and help others!
+
+* [Discussions on Github](https://github.com/mumble-voip/mumble/discussions)
+* Our [issue tracker](https://github.com/mumble-voip/mumble/issues)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
+* Our [Forums](https://forums.mumble.info/)
+<!-- forums deprecated? -->
+
+# Hosting
+
+Host your own Mumble Server and register it on our Public Server List.
+<!-- add server guide {{< wiki Running_Murmur /v}} -->
```

Comment:
```suggestion
<!-- add server guide {{< wiki Running_Murmur />}} -->
```
I think this should fix the CI

Reply:
So this little problem lead to total failure :smile: .

Reply:
Apparently - it's especially weird since it's in a comment xD